tests/test_nanoevents_physlite.py

```diff
@@ -26,6 +29,9 @@ def test_load_single_field_of_linked(events):
         events.Electrons.caloClusters.calE.compute()
 
 
+@pytest.mark.skip(
+    reason="temporarily disabled because of uproot issue #1267 https://github.com/scikit-hep/uproot5/issues/1267"
+)
```

Comment:
Do we want to wait for an uproot release with the fix?

Reply:
@SamKelson please create a tracking issue so we reactive this test when an uproot release is made!

Reply:
@jpivarski Can we poke you for a patch release for uproot with https://github.com/scikit-hep/uproot5/pull/1269 in it?

Reply:
@lgray added a tracking issue, see issue #1164